packages/core/src/utils/spanUtils.ts

```diff
@@ -280,3 +287,15 @@ export function updateMetricSummaryOnActiveSpan(
     updateMetricSummaryOnSpan(span, metricType, sanitizedName, value, unit, tags, bucketKey);
   }
 }
+
+/**
+ * Logs a warning once if `beforeSendSpan` is used to drop spans.
+ *
+ * todo(v9): Remove this once we've stopped dropping spans via `beforeSendSpan`.
+ */
+export function showSpanDropWarning(): void {
+  if (DEBUG_BUILD && !hasShownSpanDropWarning) {
+    logger.warn(SPAN_DROP_WARNING);
```

Comment:
Since this is actually an important warning I would show it even in non-debug mode. This can be done like we do it here: https://github.com/getsentry/sentry-javascript/blob/18abf39558474af40ba38587472ed43f22d52435/packages/browser/src/sdk.ts#L164-L170

It's important to put the  console call into the sandbox because not doing so would create Sentry events with the captureconsoleIntegration.

Reply:
but with a console.warn in our case right?

Reply:
yes!